recipes/neper/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "neper" %}
+{% set version = "3.5.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/rquey/neper/archive/{{ version }}.tar.gz
+  sha256: b0c24ed70baf72ed160e7663760db58caba07b5db8d4106706e8b355f36c8dfd
+
+build:
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  host:
+    - cmake
+    - libblas
+    - llvm-openmp  # [osx]
+    - gsl
+    - scotch
+    - nlopt
+    - openmp
+  run:
+    - gsl
+    - scotch
+    - nlopt
+    - openmp
+    - gmsh
+
+test:
+  commands:
+    - neper --help
+
+about:
+  home: http://neper.info/
+  license: GPL-3
```

Comment:
I believe that the SPDX form is GPL-3.0 but that can be fixed later in the feedstock.

Reply:
I initially had `license: GPL-3.0`, that's why I switched to `license: GPL-3` based on the information in conda-build. From skimming over the other PRs, `license: GPL-3.0-or-later` may the right one.

Reply:
You are right. It should be `GPL-3.0-or-later`.

Reply:
As you suggested, I can easily fix that in the feedstock later.